forge-gui/res/cardsfolder/upcoming/polukranos_reborn_polukranos_engine_of_ruin.txt

```diff
@@ -0,0 +1,23 @@
+Name:Polukranos Reborn
+ManaCost:G G G
+Types:Legendary Creature Hydra
+PT:4/5
+K:Reach
+A:AB$ SetState | Cost$ 6 WP | Defined$ Self | Mode$ Transform | SorcerySpeed$ True | AILogic$ Always | SpellDescription$ Transform CARDNAME. Activate only as a sorcery.
```

Comment:
Include `({W/P} can be paid with either {W} or 2 life.)` in SpellDescription$

Reply:
Added!